### PR TITLE
Reorder kubernetes yamls with dependencies first

### PR DESCRIPTION
**Pull request by c-nixon**
Reorders the k8s yamls so that items depended on by other items are created before their dependants.

### CODE REVIEW

File: k8s/logdna-agent.yaml [hunk 1]
```diff
@@ -8,6 +8,81 @@ metadata:
...
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: logdna-agent
+subjects:
+  - kind: ServiceAccount
+    name: logdna-agent
+    namespace: logdna-agent
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
```
Comment (mdeltito) [inline k8s/logdna-agent.yaml:70]: should this be after `ClusterRole`? here and in the others.

**Review by mdeltito — COMMENTED** [review]
one question but the yaml looks sane

File: k8s/logdna-agent.yaml (hunk 1 above)
Comment (c-nixon) [reply k8s/logdna-agent.yaml:70]: Looks like it refers to the clusterrole on line 25, it's in a different order from the other files, which is odd... It was that way before though, and the role is still defined before the binding so I think it's correct

Comment (mdeltito) [reply k8s/logdna-agent.yaml:70]: yeah seems fine as-is with the dependency order, I was thinking more in terms of organization keeping them together. no strong feelings here though

**Review by mdeltito — DISMISSED** [review]

**Review by jakedipity — APPROVED** [review]

**Review by mdeltito — DISMISSED** [review]

**Review by jakedipity — APPROVED** [review]

**Review by jakedipity — APPROVED** [review]

**Review by mdeltito — APPROVED** [review]

**Pull request merged**